extensions/adobe/experience/campaign/orchestration/experienceevent.schema.json

```diff
@@ -0,0 +1,90 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id":
+    "https://ns.adobe.com/experience/campaign/orchestration/experienceevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Journey experience event",
+  "type": "object",
+  "description": "Orchestration extension to ExperienceEvent",
+  "definitions": {
+    "journey": {
+      "title": "Journey of an entity",
+      "type": "object",
+      "description": "Journey created for an entity by marketer.",
+      "properties": {
+        "@id": {
+          "title": "Journey unique identifier",
+          "type": "string",
+          "format": "uri",
+          "description": "The unique identifier of the journey created by the marketer."
+        }
+      },
+      "required": ["@id"]
+    },
+    "journeyVersion": {
+      "title": "Journey version",
+      "type": "object",
+      "description": "Schema for describing the version of a journey, where version holds the state machine",
+      "properties": {
+        "@id": {
+          "title": "Journey version identifier",
+          "type": "string",
+          "format": "uri",
+          "description": "The unique identifier denoting the version of journey on which the entity is active."
+        }
+      },
+      "required": ["@id"]
+    },
+    "action": {
+      "title": "Action on step transition",
+      "type": "object",
+      "description": "Action taken during a step transition that happened for an entity in state machine",
+      "properties": {
+        "xdm:actionID": {
+          "title": "Action identifier",
+          "type": "string",
+          "format": "uri",
+          "description": "Unique identifier denoting the associated action."
+        },
+        "xdm:actionType": {
+          "title": "Action Type",
+          "type": "string",
+          "description": "The type of action to be performed.",
+          "meta:enum": {
```

Comment:
I am a bit at unease with the labels of this enum:

* They seem to be inconsistently named, for example _participe-substantive_ as in `scheduled_notification` or `parametrized_action` vs. _imperative-substantive_ as in `send_journey_notification` or just typed name as in `acs_writer`
* The `acs_*` labels seem very much tied into the ACS product

I think the labels should be consistently structured and not be tied to a current product name (which may change outside of our control and which limits the general applicability of this data structure.

Reply:
1)This actionType is already introduced XDM reporting event and now I am referring the same in campaign experience event. If inconsistent names needs to be handled here then it would be a breaking change. Should I create a separate issue and assign it to the concern person?
2) I agree with your comment in core XDM schema perspective. But generally xdm extension schemas are the only way to introduce product specific attributes. In that case, it should not be a problem use such names.

Reply:
@ksradobe wrote:

> This actionType is already introduced XDM reporting event and now I am referring the same in campaign experience event. If inconsistent names needs to be handled here then it would be a breaking change. 

This is a good call-out. Yet what is evolutionary state of this ? is it draft or stabilizing or finalized ?

Reply:
It is in stabilizing state.

Reply:
Yes, I like that much better.

How about

- renaming `campaign_rest_call` to `campaign_request` and
- renaming `campaign_messageService_call` to `messageService_request` ?

Reply:
@fmeschbe pushed my changes.